sensibo/StateHandler.js

```diff
@@ -132,10 +134,10 @@ module.exports = (device, platform) => {
 				const sensiboNewClimateReactState = unified.sensiboFormattedClimateReactState(device, state)
 
 				log.easyDebug(device.name, ' -> Setting New State:')
-				log.easyDebug(JSON.stringify(sensiboNewACState, null, 2))
+				log.easyDebug(JSON.stringify(sensiboNewACState, null, 0))
 
 				if (platform.enableClimateReactAutoSetup) {
-					log.easyDebug(JSON.stringify(sensiboNewClimateReactState, null, 2))
+					log.easyDebug(JSON.stringify(sensiboNewClimateReactState, null, 0))
 				}
 
 				try {
```

Comment:
Yes, because the call on line 84 only happens when you set the smartMode property (which really only happens when you enable/disable climate react from the switch). On the other hand, the code you referenced gets called every time the AC's state is change. I'll explain. Say that you updated the target temperature to 25, then climate react's range should be updated to [24, 26]. This is where this update happens. I hope what I wrote is clear, it might be a bit difficult to follow.